src/components/SearchButton.js

```diff
@@ -8,6 +8,14 @@ import { getDomElement } from '../utils';
 export const SearchButton = ({ onClick }) => {
   const { config } = useAppContext();
 
+  const inputSelector = getDomElement(config.inputContainer);
```

Comment:
The variable name `inputSelector` is misleading in this context because of the error we throw below and because it's a DOM element at this point.

What about `inputContainer` or `inputElement`?